helm/cluster-shared/templates/_psps.tpl

```diff
@@ -1,5 +1,6 @@
 {{ define "psps" }}
-{{- if .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
```

Comment:
We should still be checking for PSPs being enabled in the cluster.

Reply:
i aggree, but we currently checking on the wrong side. This chart is generated/rendered in the management cluster and we need the PSP on the workload cluster where we can't be sure that PSPs are still a thing.

Reply:
🤯  You're totally right. 🤦